resources/prosody-plugins/token/util.lib.lua

```diff
@@ -1,11 +1,108 @@
 -- Token authentication
 -- Copyright (C) 2015 Atlassian
 
+local basexx = require "basexx";
+local have_async, async = pcall(require, "util.async");
+local hex = require "util.hex";
+local http = require "net.http";
+local json = require "cjson";
+local path = require "util.paths";
+local sha256 = require "util.hashes".sha256;
+local timer = require "util.timer";
+
 local jwt = require "luajwtjitsi";
 
 local _M = {};
 
-local function _verify_token(token, appId, appSecret, disableRoomNameConstraints)
+local appId = module:get_option_string("app_id");
```

Comment:
After thinking about it some more I find it strange that utility module is loading config values. It's hard to tell which module owns this config options and where this values should be declared.